stellar/processors.js

```diff
@@ -0,0 +1,215 @@
+            const chainName = nativeToScVal(chain.axelarId, { type: 'string' });
+            const nativeTokenAddress = nativeToScVal(Address.fromString(chain?.tokenAddress), { type: 'address' });
+            const interchainTokenWasmHash = BytesToScVal(await uploadContract('InterchainToken', options, wallet, chain));
+            const tokenManagerWasmHash = BytesToScVal(await uploadContract('TokenManager', options, wallet, chain));
+
+            return {
+                owner,
+                operator,
+                gatewayAddress,
+                gasServiceAddress,
+                itsHubAddress,
+                chainName,
+                nativeTokenAddress,
+                interchainTokenWasmHash,
+                tokenManagerWasmHash,
+            };
+        }
+
+        case 'AxelarOperators':
+            return { owner };
+
+        case 'AxelarGasService': {
+            const operatorsAddress = chain.contracts?.AxelarOperators?.address;
+
+            validateParameters({
+                isValidStellarAddress: { operatorsAddress },
+            });
+
+            const operator = operatorsAddress ? nativeToScVal(Address.fromString(operatorsAddress), { type: 'address' }) : owner;
+
+            return { owner, operator };
+        }
+
+        case 'Upgrader': {
+            return {};
+        }
+
+        case 'AxelarExample': {
+            const gatewayAddress = nativeToScVal(Address.fromString(chain.contracts?.AxelarGateway?.address), { type: 'address' });
+            const gasServiceAddress = nativeToScVal(Address.fromString(chain.contracts?.AxelarGasService?.address), { type: 'address' });
+            const itsAddress = options.useDummyItsAddress
+                ? gatewayAddress
+                : nativeToScVal(chain.contracts?.InterchainTokenService?.address, { type: 'address' });
+
+            return { gatewayAddress, gasServiceAddress, itsAddress };
+        }
+
+        case 'Multicall': {
+            return {};
+        }
+
+        default:
+            throw new Error(`Unknown contract: ${contractName}`);
+    }
+};
+
+const uploadContract = async (contractName, options, wallet, chain) => {
+    const contractCodePath = await getContractCodePath(options, contractName);
+    return await uploadWasm(wallet, chain, contractCodePath);
+};
+
+const uploadWasm = async (wallet, chain, filePath) => {
+    const bytecode = readFileSync(filePath);
+    const operation = Operation.uploadContractWasm({ wasm: bytecode });
+    const wasmResponse = await broadcast(operation, wallet, chain, 'Uploaded wasm');
+    return wasmResponse.value();
+};
+
+const mainProcessor = async (options, processor, contractName) => {
+    const config = loadConfig(options.env);
+    const chain = getChainConfig(config, options.chainName);
+
+    if (!chain.contracts) {
+        chain.contracts = {};
+    }
+
+    await processor(options, config, chain, contractName);
+    saveConfig(config, options.env);
+};
+
+module.exports = {
```

Comment:
I don't see much point of moving processors into a separate file from deploy-contract.js. Even `commands.js` I don't really mind keeping in the same one since they're quite coupled together, but I can understand since it's getting large.